### ISSUE TITLE
Shorthand redirects off main website

### ISSUE THREAD

**Issue by elreydetoda** [JupiterBroadcasting/jupiterbroadcasting.com#294]
Mentioned shorthand redirectors off of main website during Office Hours 10:
- /live to https://jb-live.jupiterbroadcasting.net/
- /mumble to https://new.jupiterbroadcasting.com/community/mumble/

@ChrisLAS had mentioned he'd possibly have more as well.

**Comment by ironicbadger** [comment]
- [x] matrix
- [ ] ...?

**Comment by kd2flz** [comment]
yeah, just noticed this. The "live" link still redirect to https://jb-live.jupiterbroadcasting.net/

**Comment by gerbrent** [comment]
see #161 for a slightly more complete list...

**Comment by gerbrent** [comment]
> yeah, just noticed this. The "live" link still redirect to https://jb-live.jupiterbroadcasting.net/

I really wish someone could do #233 that would likely solve some of these.... (maybe I should learn how ; )

**Comment by kd2flz** [comment]
I'll look at #233 this evening. Not sure if it's within my skillset, but worth a shot for sure.

**Comment by elreydetoda** [comment]
> > yeah, just noticed this. The "live" link still redirect to https://jb-live.jupiterbroadcasting.net/
> 
> I really wish someone could do #233 that would likely solve some of these.... (maybe I should learn how ; ) Will make a new ticket for the above, and I'll likely get started on solving it too (trivial...)

@gerbrent I had a question I'd posted on that issue (#233), to clarify what you were wanting from it, and I was going to do it so I could also just close #300

**Comment by gerbrent** [comment]
oh sorry! question answered (hopefully satisfactorily) 

@kd2flz: ok to let @elreydetoda take #233 since he's hugo-skilled, which will likely be essential on this one?
